File: translations/codenet/deepseek_commented/Go2P/granite/Code/s144285625.py

import sys
from bisect import bisect_left, bisect_right

def main():
    n = int(input())
    ar = list(map(int, input().split()))
    ar.sort()
    print(sum(ar[bisect_right(ar, ar[i] + 1) - 1] for i in range(n)))

if __name__ == "__main__":
    main()

File: translations/codenet/deepseek_commented/Go2C++/starcoder/Code/s144285625.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	int n;
	cin >> n;
	int ar[3*n];
	for (int i = 0; i < 3*n; i++) {
		cin >> ar[i];
	}
	sort(ar, ar+3*n);
	int sum = 0;
	for (int i = 0; i < n; i++) {
		sum += ar[3*n-(i+1)*2];
	}
	cout << sum << endl;
}

